AR-builder.groovy

```diff
@@ -121,6 +121,16 @@ pipeline {
                         }
                     }
                 }
+
+                stage('Keep builds forever if important') {
+                    steps {
+                        script {
+                            if (params.SOURCE_BRANCH == "develop" || params.SOURCE_BRANCH == "release/candidate" || params.SOURCE_BRANCH == "main" || params.SOURCE_BRANCH == "prod") {
```

Comment:
can we change it to keep is the branch name starts with release/
because we can sometimes build apps form older releases

Reply:
Good point! Fixed